net/sslh/pkg-desc

```diff
@@ -0,0 +1,4 @@
+Manage SSLH, the SSL/SHH multiplexer via the OPNsense web UI.
```

Comment:
It's "pkg-descr" and can you merge the changelog into it? Usually we also use the text from the FreeBSD port description since people may not know what "sslh" is otherwise (you can see this from the GUI before install with the details button in the plugins view).